DESCRIPTION

```diff
@@ -8,12 +8,14 @@ Authors@R: c(
     person(family = "Adobe Systems Incorporated", role = c("ctb", "cph"), comment = "Source Sans Pro font")
     )
 Description: Create dashboards with 'Shiny'. This package provides
-  a theme on top of 'Shiny', making it easy to create attractive dashboards.
+    a theme on top of 'Shiny', making it easy to create attractive dashboards.
 URL: http://rstudio.github.io/shinydashboard/
 Depends:
     R (>= 3.0)
 License: GPL-2 | file LICENSE
 Imports:
     utils,
-    shiny (>= 0.11.1),
+    shiny (>= 0.12.1),
```

Comment:
Yes and no. So in the princip no. But given this https://github.com/rstudio/shinydashboard/issues/110, we want to bump it in order to fix that issue. 

(https://github.com/rstudio/shiny/tree/master/inst/www/shared - I guess 0.13 already has it). So I will bump even further to it as 1) I dont see a reason why not to do it. 2) "Addoption" of a new update of the package should be very high (similar to the one of Apple)